api/Element.json

```
@@ -8122,7 +8122,8 @@
             },
             "firefox": [
               {
-                "version_added": "59"
+                "version_added": "59",
+                "notes": "From Firefox 82 throws <code>NotFoundError</code> exception if pointer id is invalid (as per <a href='https://w3c.github.io/pointerevents/#setting-pointer-capture'>WC3 PointerEvents specification</a>). Earlier Firefox versions throw <code>InvalidPointerId</code>. See <a href='https://bugzil.la/1662124'>bug 1662124</a>."
```

So I added this to the latest full `version_added` section. Should I have added a new version  for FF82 in order to add the note to that? I think not, but can you please confirm.

Further, for firefox android, what does it mean that three is both `"version_added": false` and `"version_added": "41",`? I added the same note to the one that wasn't false.

---

If this is the specified behavior, then I think we describe the old behavior as exceptional. What do you think of inverting things like this?

```suggestion
                "notes": "Before Firefox 82, <code>setPointerCapture()</code> throws <code>InvalidPointerId</code> for an invalid <code>pointerId</code> argument. From Firefox 82, it throws <a href='https://w3c.github.io/pointerevents/#setting-pointer-capture'>the specified</a> <code>NotFoundError</code> exception. See <a href='https://bugzil.la/1662124'>bug 1662124</a>."
```

---

@ddbeck Three are arguments either way, but on balance I think you're right.

---

@ddbeck Further, is adding notes to the version_added the right thing to do? You might argue this note could belong to all versions, or be added to a new version_added entry with value 82. [ie. just confirming that doing it the way I have is "the normal way"]